astor/__init__.py

```diff
@@ -19,7 +21,8 @@
 from .op_util import symbol_data  # NOQA
 from .tree_walk import TreeWalk  # NOQA
 
-__version__ = '0.8.0'
+ROOT = Path(os.path.dirname(__file__))
+__version__ = (ROOT / 'VERSION').read_text().strip
```

Comment:
Since we now have `astor/VERSION`, I think we can drop `__version__` variable. I can't think of a use case where a project needs to support multiple astor versions at the same time.

Reply:
I would say it's pretty standard that you can do a `module.__version__`, such as: `import tensorflow; print(tensorflow.__version__)`